### PR TITLE
Added controller for API requests

### PR DESCRIPTION
**Pull request by Dalphus**
controller uses axios and dotenv to handle requests and authentication, respectively
updated dependencies
controller exports a thenable promise to use

### CODE REVIEW

**Comment by BlandSchwad** [comment]
I'm excited to make API calls

**Review by BlandSchwad — APPROVED** [review]
Looks like a good controller function

File: server/controllers.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+const axios = require( 'axios' );
+
+const api = path.join( 'https://app-hrsei-api.herokuapp.com/api/fec2/', process.env.CAMPUS_CODE );
+
+/**
+ * Makes a get request to the API using the endpoint specified
+ * @param endpoint destination in the API for the request
+ * @param config [optional] additional parameters to send with the request
+ * @returns A thenable promise to use
+ */
```
Comment (MarZmu) [inline server/controllers.js:12]: Perhaps able to condense by making line 5 a template literal and then scooting it to the top of the function that adds the endpoint to it, for clarities sake

File: spec/api.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+require( 'dotenv' ).config();
+const controllers = require('../server/controllers.js');
+
```
Comment (MarZmu) [inline spec/api.js:3]: not sure i understand the naming convention of spec for the file. Is the intention to make of of these files for each route or to do all gets in this file for all routes?

**Review by MarZmu — APPROVED** [review]
Opportunity to condense code for clarity. For the beginning of the project, might be better to be more explanatory in comments then code based (ie. explain the why the lines exist the way they do rather than what the line of code literally is, if that makes sense

**Pull request merged**